### PR TITLE
Enhancement: Run 'composer validate' as part of the build

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] runs `composer validate` as part of the build

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -2,13 +2,26 @@
 <project name="Extra strict and opinionated rules for PHPStan" default="check">
 
 	<target name="check" depends="
+		composer-validate,
 		composer,
```
Comment (localheinz) [inline build.xml]: What do you think about renaming this target to `composer-install`, for consistency?

Comment (ondrejmirtes) [reply build.xml]: Yes please :)

Comment (localheinz) [reply build.xml]: Done!

**Comment by localheinz** [comment]
Thank you, @ondrejmirtes!

**Comment by ondrejmirtes** [comment]
Awesome :)

**Pull request merged**